test/math/FloatVectorListTest.ooc

```diff
@@ -144,6 +144,23 @@ FloatVectorListTest: class extends Fixture {
 			list1 free()
 			list2 free()
 		})
+		this add("scalar product", func {
+			list1 := FloatVectorList new()
+			list2 := FloatVectorList new()
+			list1 add(1.0f)
+			list1 add(2.0f)
```

Comment:
I can change it. A general question though; even though this follows the style guide, the multiple places where this is done in the rest of the file does not do it like this. Would it always be advised to follow the style guide in this case or keep the style consistent with the rest of the file?